pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/route/GetOptionsForActivityInstanceQuestion.java

```diff
@@ -0,0 +1,113 @@
+package org.broadinstitute.ddp.route;
+
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.ddp.constants.RouteConstants;
+import org.broadinstitute.ddp.content.ContentStyle;
+import org.broadinstitute.ddp.content.I18nContentRenderer;
+import org.broadinstitute.ddp.db.TransactionWrapper;
+import org.broadinstitute.ddp.db.dao.JdbiPicklistOption;
+import org.broadinstitute.ddp.db.dao.JdbiQuestion;
+import org.broadinstitute.ddp.db.dto.LanguageDto;
+import org.broadinstitute.ddp.db.dto.PicklistOptionDto;
+import org.broadinstitute.ddp.json.RemoteAutoCompleteResponse;
+import org.broadinstitute.ddp.model.activity.instance.question.PicklistOption;
+import org.broadinstitute.ddp.security.DDPAuth;
+import org.broadinstitute.ddp.util.PicklistOptionTypeaheadComparator;
+import org.broadinstitute.ddp.util.RouteUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+import spark.Route;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+/**
+ *  This route returns the list of suggestions for the supplied Picklist Option label
+ */
+public class GetOptionsForActivityInstanceQuestion implements Route {
+
+    private static final Logger LOG = LoggerFactory.getLogger(GetOptionsForActivityInstanceQuestion.class);
+    private static final int DEFAULT_LIMIT = 100;
+    private final I18nContentRenderer renderer;
+
+    public GetOptionsForActivityInstanceQuestion(I18nContentRenderer renderer) {
+        this.renderer = renderer;
+    }
+
+    @Override
+    public RemoteAutoCompleteResponse handle(Request request, Response response) {
+        String studyGuid = request.params(RouteConstants.PathParam.STUDY_GUID);
+        String participantGuid = request.params(RouteConstants.PathParam.USER_GUID);
+        String instanceGuid = request.params(RouteConstants.PathParam.INSTANCE_GUID);
+        String questionStableId = request.params(RouteConstants.PathParam.QUESTION_STABLE_ID);
+
+        DDPAuth ddpAuth = RouteUtil.getDDPAuth(request);
+        String operatorGuid = StringUtils.defaultIfBlank(ddpAuth.getOperator(), participantGuid);
+        boolean isStudyAdmin = ddpAuth.hasAdminAccessToStudy(studyGuid);
+        String autoCompleteQuery = request.queryParams(RouteConstants.QueryParam.TYPEAHEAD_QUERY);
+        String queryLimit = request.queryParams(RouteConstants.QueryParam.TYPEAHEAD_QUERY_LIMIT);
+        int limit = StringUtils.isNotBlank(queryLimit) ? Integer.valueOf(queryLimit) : DEFAULT_LIMIT;
```

Comment:
How about just sticking to default as MAX limit too if the ask is more than default ?
Trying to avoid erroring out and sticking to default silently. 
If needed we can update APIDocs that max results will be default value of 100